### PR TITLE
Fixing mock-data script issue on windows

### PR DESCRIPTION
**Pull request by ZaradarBH**
## Hey, I just made a Pull Request!

Fix for https://github.com/spotify/backstage/issues/1557

#### :heavy_check_mark: Checklist

- [x] All tests are passing `yarn test`
- [x] Screenshots attached (for info)
- [ ] Relevant documentation updated
- [x ] Prettier run on changed files
- [ ] Tests added for new functionality
- [ ] Regression tests added for bug fixes

mock-data-on-windows

### CODE REVIEW

**Review by Rugvip — COMMENTED** [review]
Awesome, thanks! 😁

Better keep the file permissions intact for the scripts though, unix-based systems will prolly stop working otherwise. This PR changes them from 755 to 644 atm. See 2nd answer in https://stackoverflow.com/questions/25730041/updating-file-permissions-with-git-bash-on-windows-7 for how to do that if Windows is causing trouble

**Comment by ZaradarDFDS** [comment]
Being a windows dev in a world that is moving more towards linux can be a pain 🤣 I am still moving into the code base and didnt even consider it would do that because I am used to running everything in containers, I will fix it in the morning and update the PR.
Thx for the heads up and hope your kicking ass in LoL 🥊

**Comment by SEQUOIIA** [comment]
The file permissions should be better now 😃

**Comment by ZaradarDFDS** [comment]
Thanks Emil. :)

File: ADOPTERS.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
-| Organization | Contact | Description of Use |
-| ------------ | ------- | ------------------ |
-| [Spotify](https://www.spotify.com) |[@alund](https://github.com/alund)| Main interface towards all of Spotify's infrastructure and technical documentation.|
+| Organization                       | Contact                                    | Description of Use                                                                  |
+| ---------------------------------- | ------------------------------------------ | ----------------------------------------------------------------------------------- |
+| [Spotify](https://www.spotify.com) | [@alund](https://github.com/alund)         | Main interface towards all of Spotify's infrastructure and technical documentation. |
+| [DFDS](https://www.dfds.com)       | [@carlsendk](https://github.com/carlsendk) | Internal self-service at DFDS.                                                      |
```
Comment (Rugvip) [inline ADOPTERS.md:4]: Please make this change in a separate PR to not mix up discussions.

**Review by Rugvip — COMMENTED** [review]
Thanks for the fix! Please do split the adopters change into a separate PR though 😅

File: ADOPTERS.md (hunk 1 above)
Comment (ZaradarDFDS) [reply ADOPTERS.md]: Reverted :) We will re-introduce it in a later PR, we are just very eager to show our support for this great idea ;)

**Review by Rugvip — APPROVED** [review]
👌

**Comment by stefanalund** [comment]
Thanks a lot! Adding yourselves to ADOPTERS helps us a lot 🙏💪🏼
